app/uploaders/profile_photo_uploader.rb

```diff
@@ -0,0 +1,47 @@
+class ProfilePhotoUploader < CarrierWave::Uploader::Base
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  # include CarrierWave::MiniMagick
+
+  # Choose what kind of storage to use for this uploader:
+  storage :file
+  # storage :fog
+
```

Comment:
you do such a good job of configuring fog/aws.....and then you dont actually tell your uploader to use it! go back back to the cloud storage article and see the instruction there. We want to use file storage in test and fog in dev and production